### DISCUSSION TITLE
[Menubar] Open first-time on hover instead of on-click

### DISCUSSION THREAD

**Discussion by mbaldas** [radix-ui/primitives#2245] [Help]
## Question

Hello everyone,
I'm currently building a "topbar" menu using the [`Menubar`](https://www.npmjs.com/package/@radix-ui/react-menubar) component.

## Behavior

I have all the menu items and I need to **click** in one of them to open the content box for the **first time** and then on the other interactions they all can be opened by hovering.

## Expected Behavior

I have all the menu items and they don't need to be clicked to open on the first time, all of them can be opened direclty through a hover action.

_I've tried to use the `NavigationMenu`, but for my use-case it feels more correct to use the `Menubar`one_.

Appreciate any sort of help,
Thank you :)

**Comment by joaom00** [comment]
Hi @mbaldas, you can control the `Menubar` with `value` and `onValueChange` and then open when the pointer enter in the `Trigger`

**Reply by alekseykurylev** [reply]
Could you give an example?
